Std/Data/List/Lemmas.lean

```diff
@@ -4,9 +4,11 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Parikshit Khanna, Jeremy Avigad, Leonardo de Moura, Floris van Doorn, Mario Carneiro
 -/
 import Std.Control.ForInStep.Lemmas
+import Std.Data.Nat.Lemmas
```

Comment:
I'm unhappy about adding this import. Can whatever relies on this move to a different/new file?